onnxscript/function_libs/torch_aten/ops/special.py

```diff
@@ -205,10 +206,12 @@ def aten_special_log_ndtr(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_special_log_softmax(self: TensorType, dim: int, dtype: int = -1) -> TensorType:
+@torch_op("aten::log_softmax")
+def aten_special_log_softmax(self: TTensor, dim: int, dtype: int = FLOAT.dtype) -> TTensor:
```

Comment:
Just to confirm, does onnx logsoftmax support all tensor types?

Reply:
good suggestion. changed to TFloatOrBFloat16